BioArchLinux/jasp-desktop/PKGBUILD

```diff
@@ -1,9 +1,9 @@
 #Maintainer: sukanka <su975853527 AT gmail.com>
 
 _pkgname=jasp
-_pkgver=0.18.2.beta
+_pkgver=0.18.2
```

Comment:
Note: this will cause a downgrade!.
as reported by `vercmp`: `0.18.2.beta` is newer than `0.18.2`.
even if you downgrade it to `0.18.2` manually, lilac will update it to `0.18.2.beta` next time.

So a better way is to file an issue to the upstream, asking for a newer release, `0.18.2.0` (which can have the same content as `0.18.2`)

Reply:
We can set `epoch=1` in the PKGBUILD, which makes the `0.18.2` version newer. Also it would be better to use `use_latest_release` instead of `use_max_tag` in the nvchecker config, as that will exclude any pre-releases.

Reply:
> We can set `epoch=1` in the PKGBUILD, which makes the `0.18.2` version newer. 

yeah, but lilac will update `pkgver` to `0.18.2.beta` as `0.18.2.beta >0.18.2` (and `epoch` remains 1)

each time upstream does this, we will have to bump `epoch` several times.

So, I prefer to skip `0.18.2` if there is no `0.18.2.0`

>  Also it would be better to use use_latest_release instead of use_max_tag in the nvchecker config, as that will exclude any pre-releases.

I don't want to exclude any pre-release.

Reply:
Well, thanks for details so what should I do? but the 0.18.2 was released and we may not change version number to 0.18.2.x cuz JASP is a cross-platform software and four digit was not respected on Windows installer.

Yeah, `0.18.2.beta` is same as `0.18.2` due to it's a pre-release version for _Release Candidate_ .

I will drop the version number commit then.

谢谢两位。